src/entity/MemberTask.java

```diff
@@ -0,0 +1,22 @@
+package entity;
+
+import java.util.Date;
+
+public class MemberTask {
+    public String owner;
```

Comment:
```suggestion
    public String memberEmail;
```

Reply:
This needs to be the unique identifier that get's a member's task. For now, we will assume members have only one task at a time. ProjectStateManager should keep a List<MemberTask>.